aiida_vasp/io/outcar.py

```diff
@@ -1,30 +1,65 @@
 """Tools for parsing OUTCAR files."""
 
+from aiida_vasp.utils.aiida_utils import get_data_class
+from aiida_vasp.io.parser import BaseFileParser
 
-class OutcarParser(object):
-    """parse OUTCAR into a dictionary, which is supposed to be turned into ParameterData later."""
+DEFAULT_OPTIONS = {'quantities_to_parse': ['volume', 'energies', 'efermi']}
 
-    def __init__(self, fname):
+
+class OutcarParser(BaseFileParser):
```

Comment:
Please add unit tests for this class in aiida_vasp/io/tests/test_outcar_io.py

Reply:
Added test for OutcarParser